synchro/__init__.py

```diff
@@ -358,6 +367,12 @@ def is_locked(self):
         # # Use the property directly from the underlying MongoClient.
         return self.delegate.delegate.is_locked
 
+    def _duplicate(self, **kwargs):
+        client = self.delegate._duplicate(**kwargs)
```

Comment:
Can we add a brief comment here saying that `_duplicate` is expected to return a regular MongoClient?

Reply:
Done.